scripts/debian-setup.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+RED='\033[1;31m'
+NC='\033[0m' # No Color
+
+echo -e "${RED}Adding arm user${NC}"
+groupadd arm
+useradd -m arm -g arm -G cdrom
+passwd arm
+echo -e "${RED}Installing git${NC}"
+apt-get install git
+echo -e "${RED}Installing required build tools${NC}"
+apt-get install build-essential pkg-config libc6-dev libssl-dev libexpat1-dev libavcodec-dev libgl1-mesa-dev qtbase5-dev zlib1g-dev
+echo -e "${RED}Installing wget${NC}"
+apt-get install wget
+
+echo -e "${RED}Setting up directories and getting makeMKV files${NC}"
+mkdir /makeMKV
+cd /makeMKV
+
+wget https://www.makemkv.com/download/makemkv-bin-1.15.4.tar.gz
+wget https://www.makemkv.com/download/makemkv-oss-1.15.4.tar.gz
+
+echo -e "${RED}Extracting MakeMKV${NC}"
+tar xvzf makemkv-oss-1.15.4.tar.gz
+tar xvzf makemkv-bin-1.15.4.tar.gz
+
+cd makemkv-oss-1.15.4
+echo -e "${RED}Installing MakeMKV${NC}"
+./configure
+make
+make install
+
+cd ../makemkv-bin-1.15.4
+make
+make install
+
+echo -e "${RED}Installing ffmpeg${NC}"
+apt install ffmpeg
+
+echo -e "${RED}Installing ARM requirments${NC}"
+apt install handbrake-cli libavcodec-extra
+apt install libdvdcss2
+apt install abcde flac imagemagick glyrc cdparanoia
+apt install at
+apt install python3 python3-pip
+apt install libcurl4-openssl-dev libssl-dev  
+apt install libdvd-pkg
+wget http://download.videolan.org/pub/debian/stable/libdvdcss2_1.2.13-0_amd64.deb
+wget http://download.videolan.org/pub/debian/stable/libdvdcss_1.2.13-0.debian.tar.gz
+wget http://ftp.us.debian.org/debian/pool/contrib/libd/libdvd-pkg/libdvd-pkg_1.4.0-1-2_all.deb
+sudo dpkg -i libdvdcss2_1.2.13-0_amd64.deb
+sudo dpkg -i libdvd-pkg_1.4.0-1-2_all.deb
+apt --fix-broken install
+dpkg-reconfigure libdvd-pkg
+apt install default-jre-headless
+apt install eject
+
+echo -e "${RED}Installing ARM:Automatic Ripping Machine${NC}"
+cd /opt
+mkdir arm
+chown arm:arm arm
+chmod 775 arm
+git clone https://github.com/automatic-ripping-machine/automatic-ripping-machine.git arm
+chown -R arm:arm arm
+cd arm
+pip3 install setuptools
+apt-get install python3-dev python3-pip python3-venv python3-wheel -y
+pip3 install wheel
+pip3 install -r requirements.txt 
+ln -s /opt/arm/setup/51-automedia.rules /lib/udev/rules.d/
+ln -s /opt/arm/setup/.abcde.conf /home/arm/
+cp docs/arm.yaml.sample arm.yaml
+mkdir /etc/arm/
+ln -s /opt/arm/arm.yaml /etc/arm/
+
+mkdir -p /mnt/dev/sr0
```

Comment:
This might be something that should eventually become parameter based, but I think we should leave that until someone complains.

Reply:
This is possible in bash, I wasnt sure what the best default action was so i left it as simple as possible.